engine/src/flutter/lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -1790,10 +1790,10 @@ extension type SkParagraphBuilderNamespace(JSObject _) implements JSObject {
   );
 
   bool RequiresClientICU() {
-    if (!js_util.hasProperty(this, 'RequiresClientICU')) {
+    if (!hasProperty('RequiresClientICU'.toJS).toDart) {
```

Comment:
`has('RequiresClientICU')` for a shorthand here and everywhere else.

Reply:
yep